nestkernel/nestmodule.cpp

```diff
@@ -2155,24 +2153,24 @@ NestModule::GetPosition_gFunction::execute( SLIInterpreter* i ) const
 }
 
 /** @BeginDocumentation
-  Name: topology::Displacement - compute displacement vector
+  Name: nest::Displacement - compute displacement vector
 
   Synopsis: layer from_node_id to_node_id Displacement -> [double vector]
             layer from_pos to_node_id Displacement -> [double vector]
 
   Parameters:
   layer       - NodeCollection for layer
-  from_node_id    - int, node_id of node in a topology layer
+  from_node_id    - int, node_id of node in a spatial layer
```

Comment:
Maybe use 'NodeColletion' or 'network' here instead of layer? I think layer made sense when we used 'topology', but now that we change to 'spatial' I think it makes more sense to change the 'layer' as well. Also further down in this file.